.gitignore

```diff
@@ -1,3 +1,4 @@
 book
 tests/src/*
+tests/target/*
```

Comment:
Remove the trailing `*` please (I use a sym-link).

All paths should start with a `/` ideally.